dcos_installer/test_backend.py

```diff
@@ -345,6 +349,52 @@ def aws_cf_configure(s3_bucket_name, config, config_aws, tmpdir, monkeypatch):
         s3_bucket.delete()
 
 
+def test_do_configure_valid_config_no_duplicite_logging(tmpdir, monkeypatch, caplog):
+    """
+    Log messages aren't printed twice.
+    """
+    monkeypatch.setenv('BOOTSTRAP_VARIANT', 'test_variant')
+    create_config(simple_full_config, tmpdir)
+    create_fake_build_artifacts(tmpdir)
+    with tmpdir.as_cwd():
+        assert backend.do_configure(config_path='genconf/config.yaml') == 0
+
+    # The message comes from gen.get_dcosconfig_source_target_and_templates()function
+    expected_message = 'Generating configuration files...'
+    filtered_messages = [rec.message for rec in caplog.records if rec.message == expected_message]
+    assert [expected_message] == filtered_messages
+
+
+def test_do_configure_prints_validation_errors(tmpdir, monkeypatch, caplog):
+    """
+    Configuration validation errors are printed as `error` messages.
```

Comment:
That logs are "print"ed is, I think, a poor choice of wording from the implementation `print_messages`. Let's not keep that bad terminology in the tests. Instead, use "logged". Note that this applies to the test function name as well.